src/Symfony/Component/Message/ContainerHandlerLocator.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Message;
+
+use Psr\Container\ContainerInterface;
+use Symfony\Component\Message\Exception\NoHandlerForMessageException;
+
+/**
+ * @author Miha Vrhovnik <miha.vrhovnik@gmail.com>
+ * @author Samuel Roze <samuel.roze@gmail.com>
+ */
+class ContainerHandlerLocator implements HandlerLocatorInterface
+{
+    /**
+     * @var ContainerInterface
+     */
```

Comment:
I will probably say something stupid, but I thought IDEs were able to infer that because of the constructir.